src/com/theironyard/novauc/Facebook.java

```diff
@@ -0,0 +1,220 @@
+package com.theironyard.novauc;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+
+public class Facebook {
+
+    //Menu page for of facebook very simple menu
+    public void mainPage() throws IOException {
+        System.out.println("##############################################################");
+        System.out.println("################                               ###############");
+        System.out.println("########                   WELCOME                    ########");
+        System.out.println("######                                                  ######");
+        System.out.println("########                  TO FACEBOOK                 ########");
+        System.out.println("################                              ################");
+        System.out.println("##############################################################");
+        System.out.println("The only option available is the [ Friends ] menu.");
+        String onlyOption = Main.kb.nextLine();
+        if (onlyOption.equalsIgnoreCase("friends")) {
```

Comment:
Great job Alvaro.  I like how you have fun with the assignments and put your own twist on them.